demo.py

```diff
@@ -43,5 +43,9 @@ def run_demo():
     # ipsec stats
     print(primary_asa.ipsec_stats())
 
+    # show logging buffered
+    print(primary_asa.show_logging_buffered())
```

Comment:
This doesn't currently return anything due to the error noted above.
```
{"text": []}
```
Once you've corrected that problem the output should look something like this:
```
{"text": ["Aug 16 2017 15:35:37 KP-systest-admin : %ASA-4-711004: Task ran for 114 msec, Process = Unicorn Admin Handler, PC = f34bf8f4, Call stack = ", "Aug 16 2017 15:36:12 KP-systest-admin : %ASA-4-711004: Task ran for 108 msec, Process ...
```